pkg/utils/utils.go

```diff
@@ -22,10 +22,10 @@ package utils
  * Please add functional style container operations sparingly and intentionally.
  */
 
-func Filter(origin map[string]string, predicate func(string) bool) map[string]string {
```

Comment:
Instead of diffing maps, can we pass in a better predicate like Contains.

We can call something like:
```
Filter(map, func(key string) bool {
  return utils.Includes(DisallowedAnnotationsList, key)
})
```